webui/frontend/src/assets/main.css

```
@@ -281,111 +230,31 @@ textarea.transition-colors {
-.dark ::-webkit-scrollbar-thumb:hover {
-  background: #9ca3af;
+  background: var(--color-scrollbar-thumb-hover);
 }
 
 /* ==================== Text Selection ==================== */
 ::selection {
-  background-color: rgba(59, 130, 246, 0.25);
+  background-color: var(--color-selection);
 }
 
 ::-moz-selection {
```

Stylelint (17.14.0)

[error] Vendor-prefixed selector "::-moz-selection" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)